extensions/kadena/src/extension.ts

```diff
@@ -0,0 +1,147 @@
+import { ITatumSdkContainer, Network, TatumConfig, TatumSdkExtension } from '@tatumio/tatum'
+import {
+  ChainId,
+  createClient,
+  IUnsignedCommand,
+  Pact,
+  PactReference,
+  readKeyset
+} from "@kadena/client";
+
+import { KadenaGuard, KadenaNetworkId } from "./types";
+import {
+  COIN_GAS_CAPABILITY,
+  CREATE_TOKEN_ID_POLICIES,
+  NFT_PRECISION,
+  MARMALADE_CREATE_TOKEN_CAPABILITY,
+  MARMALADE_MINT_CAPABILITY,
+  NFT_AMOUNT,
+  MARMALADE_BURN_CAPABILITY,
+  NFT_BURN_AMOUNT,
+  MARMALADE_BURN_POLICY
+} from "./consts";
```

Comment:
I know it is technically outside of the scope, but it would be nice to also have `coin.transfer` since being able to transfer native currency is a very basic use case you need to be able to do anything else. We can discuss tomorrow, I already have the implementation ready on my side for the upcoming faucet.

Reply:
we can discuss